src/App.css

```diff
@@ -0,0 +1,83 @@
+.nav-bg {
+    background-color: #000080;
+}
+.nav-text {
+    color: white;
+}
+.footer-bg {
+    background-color: #000080;
+    color: white;
+    text-align: center;
+  }
+  header.uvp {
```

Comment:
What is uvp?

Reply:
This is the Unique Value Proposition that is shown on the landing page